include/albatross/src/models/sparse_gp.hpp

```diff
@@ -610,6 +610,14 @@ class SparseGaussianProcessRegression
     return inducing_point_strategy_;
   }
 
+  GrouperFunction get_grouper_function() const {
+    return independent_group_function_;
+  }
+
+  GrouperFunction &get_grouper_function() {
+    return independent_group_function_;
```

Comment:
I can't exactly add a `setter` since `GrouperFunction` is a template type with no information, but we can enable the caller to modify it directly

Reply:
Why does `void set_grouper_function(GrouperFunction &&) { ... }` not work?  What is the problem with using the template parameter as an argument type rather than a return type?

Reply:
I guess that should be ok actually